create_outletmn.cpp is not in src directory

Hello
I having a trouble during cmake configuration.

CMake Error at CMakeLists.txt:91 (add_executable):
  Cannot find source file:

    create_outletmn.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx
 
I downloaded the master release in the github repository, and the create_outletmn.cpp is not the src directory.
Do I have to comment the line in the cmake file where create_outletmn.cpp file is require? Or how can I solve the problem?

Thanks

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -42,7 +42,6 @@ set (GAGEWATERSHED gagewatershedmn.cpp gagewatershed.cpp ${common_srcs} ${shape_
 set (TWI TWImn.cpp TWI.cpp ${common_srcs})
 set (CONNECTDOWN ConnectDownmn.cpp ConnectDown.cpp ${common_srcs} ${shape_srcs})
 set (CATCHOUTLETS CatchOutletsmn.cpp CatchOutlets.cpp ${common_srcs} ${shape_srcs})
-set (CREATEOUTLET create_outletmn.cpp create_outlet.cpp ${common_srcs})
 set (EDITRASTER EditRastermn.cpp EditRaster.cpp ${common_srcs})
 set (FLOWDIRCOND flowdirconditionmn.cpp flowdircond.cpp ${common_srcs})
 set (RETLIMFLOW RetLimFlowmn.cpp RetlimFlow.cpp ${common_srcs})
@@ -88,7 +87,6 @@ add_executable (gagewatershed ${GAGEWATERSHED})
 add_executable (twi ${TWI})
 add_executable (connectdown ${CONNECTDOWN})
 add_executable (catchoutlets ${CATCHOUTLETS})
-add_executable (createoutlet ${CREATEOUTLET})
 add_executable (editraster ${EDITRASTER})
 add_executable (flowdircond ${FLOWDIRCOND})
 add_executable (retlimflow ${RETLIMFLOW})
@@ -124,7 +122,6 @@ set (MY_TARGETS aread8
                 twi
                 connectdown
                 catchoutlets
-                createoutlet
                 editraster
                 flowdircond
                 retlimflow

